themes/default/content/docs/intro/vs/terraform.md

```diff
@@ -11,94 +11,189 @@ aliases:
 - /docs/reference/vs/terraform/
 ---
 
-Terraform and Pulumi hold a lot of similarities, but they differ in a few key ways. This page helps provide a rundown of the differences. First, Pulumi is like Terraform, in that you create, deploy, and manage [infrastructure as code]({{< relref "/what-is/what-is-infrastructure-as-code" >}}) on any cloud. But where
-Terraform requires the use of a custom programming language, Pulumi allows you to use familiar general purpose languages and tools to accomplish the same goals. Like Terraform, Pulumi is
-[open source on GitHub](https://github.com/pulumi/pulumi) and is [free to use]({{< relref "/docs/get-started" >}}).
+Choosing the right cloud engineering platform is critical. We want you to have the best information possible to pick the solution that fits your needs.
 
-Both Terraform and Pulumi support many cloud providers, including AWS, Azure, and Google Cloud,
-plus other services like CloudFlare, Digital Ocean, and more. Thanks to integration with Terraform providers, Pulumi
-is able to support a superset of the providers that Terraform currently offers.
+## What is Terraform?
 
-Here is a summary of the key differences between Pulumi and Terraform:
+Terraform provides open-source [infrastructure as code]({{< relref "/what-is/what-is-infrastructure-as-code" >}}) software for cloud service management with a consistent CLI workflow. Terraform allows you to write, plan, and apply changes to deliver infrastructure as code.
+
+## Pulumi vs. Terraform: Similarities
```

Comment:
i know this is a draft, so feel free to ignore it if you arent interested in addressing or hearing this yet. but wanted to share early in case it would be helpful.

we moved away from the vs. language because it was not clear to many folks what vs. meant. we opted to use compare to language instead. we had to keep the page titles and links as is because there were SEO concerns.

would you be open to using language like "Similarities between Pulumi and Terraform" or similar instead?

this also potentially aligns with your message about giving folks the information they need to make the best decision for their situation. so instead of putting them against each other with the vs language, sharing information about them.

related pr: https://github.com/pulumi/pulumi-hugo/issues/560

Reply:
Most of this document has nothing to do with the similarities but rather the differences. So Pulumi vs. Terraform:  Comparison is probably more apt.

Reply:
@aaronkao the intent of my comment had nothing to do with similarities. if the section i commented on is about differences then adjusting to the language "Differences between Pulumi and Terraform" is fine. The concern here is that its not clear what we are trying to communicate by using the abbreviation "vs.".

Reply:
ah, that makes sense. thank you

Reply:
Let me check with Lindsay since this is one of the most trafficked pages through SEO and people searching for Pulumi vs Terraform